### DISCUSSION TITLE
Can I force MM to generate a MMConfigCache, ignoring any errors?

### DISCUSSION THREAD

**Discussion by ProgrammerFailure** [KSP-ModularManagement/ModuleManager#25] [Support]
As per the title, I would like to force MM to make MMConfigCache, with any errors during patching being ignored for caching purposes. Is it possible?

**Comment by Lisias** [comment]
Yes, it's possible. I don't know if it's desirable, however.

**Reply by Lisias** [reply]
As a matter of fact, checking the Source, I remembered that something like this is already implemented - but somewhat cumbersome to activate.

Add `-mm-dump` to KSP's command line and it will dump all configs into a file called `_MMCfgOutput` on `<KSP>/PluginData/ModuleManager` (the same place were I put "my" ConfigCache.

This is what you are talking about, perhaps?

**Reply by ProgrammerFailure** [reply]
The ksp command line you refer to is the one that comes up with Alt-F12 right? Will check that out soon.

**Reply by Lisias** [reply]
Yep. But without reloading everything in memory.

As a matter of fact... 

https://github.com/KSP-ModularManagement/ModuleManager/commit/f790d20d2dc753fd831f813ef6ad8fc51475a8aa
